luarules/gadgets/unit_inherit_creation_xp.lua

```diff
@@ -0,0 +1,63 @@
+function gadget:GetInfo()
+	return {
+		name = "Inherit Creation Units XP",
+		desc = "Allows units with added UnitDefs to gain a defined fraction of the XP earned by their creations.",
+		author = "SethDGamre and Xehrath",
+		date = "May 2024",
+		license = "Public domain",
+		layer = 50,
```

Comment:
why 50?

Reply:
Same as above
"unit_explosion_spawner is on that layer and so is unit_evolution gadget. I don't know how layers work, but I just read something written by HoboJoe about how it's good practice to start with a higher number and move it down if it needs prioritization. I don't know if it needs prioritization yet."

Reply:
moved to layer 0 as requested